planner/implementation/simple_plans.go

```diff
@@ -28,6 +28,12 @@ func NewProjectionImpl(proj *plannercore.PhysicalProjection) *ProjectionImpl {
 	return &ProjectionImpl{baseImpl{plan: proj}}
 }
 
+// CalcCost implements Implementation CalcCost interface.
+func (projection *ProjectionImpl) CalcCost(outCount float64, children ...memo.Implementation) float64 {
+	proj := projection.plan.(*plannercore.PhysicalProjection)
+	return proj.GetCost(children[0].GetPlan().Stats().RowCount) + children[0].GetCost()
```

Comment:
We need to set `impl.cost = proj.GetCost(children[0].GetPlan().Stats().RowCount) + children[0].GetCost()`
and return `impl.cost`

Reply:
updated